suite-native/module-accounts-management/src/components/AccountSettingsRemoveCoinButton.tsx

```diff
@@ -20,13 +14,18 @@ import {
 } from '@suite-native/navigation';
 import { useAlert } from '@suite-native/alerts';
 
-export const AccountSettingsRemoveCoinButton = ({ accountKey }: { accountKey: AccountKey }) => {
+type AccountSettingsRemoveCoinButtonProps = {
+    accountKey: AccountKey;
```

Comment:
Q: Does `AccountKey` reflects also device? Because we can have same account imported and also remembered from some device

Reply:
Not account key AFAIK. But account has a deviceState property which we can use for mapping. 

However if I remember correctly, this was already discussed within the team and decided that this WILL be possible and is okay to have account imported as well as within device, since you will never see them side by side since they're in a different portfolio.

Reply:
Okay, turns out account key has a deviceState inside the key, so we could possibly use that, if needed.